capstone/src/test/java/com/google/sps/servlets/ExploreServletTest.java

```diff
@@ -0,0 +1,129 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalURLFetchServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.common.collect.ImmutableList;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+/** */
+@RunWith(JUnit4.class)
+public final class ExploreServletTest {
+
+  private static final String KEVIN = "kshao@google.com";
+  private static final String MEGHA = "kakm@google.com";
+  private static final String MEGAN = "meganshi@google.com";
+
+  private static final String CLUB_1 = "Club 1";
+  private static final String CLUB_2 = "Club 2";
+  private static final String CLUB_3 = "Club 3";
+
+  private static final String SITE_1 = "www.fakesite.com";
+  private static final String SITE_2 = "www.realfakesite.com";
+
+  private static final String DESCRIPTION_1 = "Helping people.";
+  private static final String DESCRIPTION_2 = "Not helping people.";
+
+  private ExploreServlet servlet;
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+  private DatastoreService datastore;
+
+  private Entity club1;
+  private Entity club2;
+
+  private LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig(),
+          new LocalUserServiceTestConfig(),
+          new LocalURLFetchServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    this.servlet = new ExploreServlet();
+    MockitoAnnotations.initMocks(this);
+    helper.setUp();
+    this.datastore = DatastoreServiceFactory.getDatastoreService();
+
+    club1 = new Entity(Constants.CLUB_PROP);
```

Comment:
Chris mentioned this in one of my previous PRs (https://github.com/googleinterns/step179-2020/pull/35#discussion_r463740211)
tl;dr: It's better to define an entity and add it to Datastore inside the test method you are using it for instead of in setUp